cmd/ore/do/create-image.go

```diff
@@ -35,7 +35,7 @@ func init() {
 	DO.AddCommand(cmdCreateImage)
 	cmdCreateImage.Flags().StringVar(&options.Region, "region", "sfo2", "region slug")
 	cmdCreateImage.Flags().StringVarP(&imageName, "name", "n", "", "image name")
-	cmdCreateImage.Flags().StringVarP(&imageURL, "url", "u", "", "image source URL (e.g. \"https://stable.release.flatcar-linux.net/amd64-usr/current/flatcar_production_digitalocean_image.bin.bz2\"")
+	cmdCreateImage.Flags().StringVarP(&imageURL, "url", "u", "", "image source URL (e.g. \"https://stable.release-linux.flatcar.net/amd64-usr/current/flatcar_production_digitalocean_image.bin.bz2\"")
```

Comment:
```suggestion
	cmdCreateImage.Flags().StringVarP(&imageURL, "url", "u", "", "image source URL (e.g. \"https://stable.release.flatcar-linux.net/amd64-usr/current/flatcar_production_digitalocean_image.bin.bz2\"")
```

Reply:
Ok, should be good now - I grepped on `release-linux`, sorry for the confusion :D